early-attack-detector-py/src/utils.py

```diff
@@ -116,4 +120,41 @@ def alert_count(chain_id: int, alert_id: str) -> int:
     except Exception as err:
         logger.error(f"Error obtaining alert counts: {err}")
 
-    return alert_id_counts, alert_counts
\ No newline at end of file
+    return alert_id_counts, alert_counts
+
+def get_tp_attacker_list() -> list:
+    res = requests.get('https://raw.githubusercontent.com/forta-network/starter-kits/main/early-attack-detector-py/tp_list.csv')
+    logging.info(f"Made request to fetch fp list: {res.status_code}")
+    content = res.content.decode('utf-8') if res.status_code == 200 else open('tp_list.csv', 'r').read()
+
+    df_fps = pd.read_csv(io.StringIO(content), sep=',')
+    attacker_list = set(df_fps['Attacker'].tolist())
+
+    eth_address_pattern = re.compile(r'^0x[a-fA-F0-9]{40}$')
+
+    unique_attacker_addresses = set()
+    # Could be populated by random strings
+    # and non-Ethereum addresses
+    non_eth_entries = []
+
+    for entry in attacker_list:
+        # some values were sneaking in as `float` types
+        # (e.g. 'nan')
+        if isinstance(entry, str):
+            parts = [part.strip() for part in entry.split(',')]
+            for part in parts:
+                if eth_address_pattern.match(part):
+                    unique_attacker_addresses.add(part)
```

Comment:
```suggestion
                    unique_attacker_addresses.add(part.lower())
```

(There were some duplicates, where one is a checksum address and the other a lowercase one)